internal/cmdutil/rand.go

```diff
@@ -44,5 +44,6 @@ func SeedArgFor(bc *buildconfig.BuildConfig, imgTypeName, distributionName, arch
 	h.Write([]byte(imgTypeName))
 	h.Write([]byte(bc.Name))
 
+	// nolint:gosec
```

Comment:
Fwiw, I think this is fine, at this level we only need predictable bits for the RNG, the fact that it overflows does not matter, the number itself has no meaning (we could make it uint64 in the future but that is a lot of busywork without reallying giving us much)